server/aws-lsp-codewhisperer/src/language-server/chat/chatSessionService.ts

```diff
@@ -151,11 +151,19 @@ export class ChatSessionService {
             try {
                 return await client.generateAssistantResponse(request, this.#abortController)
             } catch (e) {
+                if (isRequestAbortedError(e)) {
+                    const requestId =
+                        e instanceof CodeWhispererStreamingServiceException ? e.$metadata?.requestId : undefined
```

Comment:
out of curiosity, should this always be defined?
Is it just the SDK generated types that allow for this to be missing?

Reply:
Ideally it should always be there. I can probably remove this check in a follow-up